x-pack/plugins/alerting/server/alert/create_alert_factory.ts

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
Github did not recognize this renaming from alert_instance/create_alert_instance_factory.ts to alert/create_alert_factory.ts.

Reply:
for future reference; I'm thinking a `git mv` can help "fix" these, compared to hoping that git catches it when you rename manually.  Though it's probably not worth the effort if it's just a few files ... (and too late now, probably?)